### ISSUE TITLE
VSCode extension: incorrect formatting

### ISSUE THREAD

**Issue by wdscxsj** [a-h/templ#192]
In VSCode v1.82.2 on Windows 10 with templ-vscode v0.0.14, executing the Format Document command repeatedly may mangle the file:

![demo](https://github.com/a-h/templ/assets/863369/7405d7bb-9697-4bec-adb7-d919932a3a6a)

Running `templ fmt` on this file repeatedly doesn't have this issue.

**Comment by wdscxsj** [comment]
Hello, after a sleep, the first issue occurs again (tearing `name` apart). Here is the exact [file](https://github.com/a-h/templ/files/12788549/hello.zip).

And it seems `templ fmt` (not the VSCode extension) doesn't work quite well with a block of commented out code. For example, it formats this

```go
// package main

// templ Hello() {
// 	<div>Hello</div>
// }

package main

templ Hello(name string) {
	<div>Hello, {name}</div>
}
```

into

```go
package main

// package main

// templ Hello() {
// 	<div>Hello</div>
// }

package main

templ Hello(name string) {
	<div>Hello, { name }</div>
}


```

**Comment by wdscxsj** [comment]
@a-h I can confirm this issue has been fixed in v0.2.408. Thanks for your wonderful work!

**Comment by a-h** [comment]
Glad to hear that it's sorted!
